dcap-libs/action.yaml

```diff
@@ -12,10 +12,17 @@ runs:
     - name: Installing DCAP libraries
       shell: bash
       run:
-        DCAP_VERSION=${{ inputs.version }}
+        DCAP_VERSION=${{ inputs.version }} &&
         source /etc/lsb-release &&
         echo "deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu $DISTRIB_CODENAME main" | sudo tee /etc/apt/sources.list.d/intel-sgx.list &&
         wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add - &&
         sudo apt-get update &&
-        sudo apt-get install -y libsgx-dcap-ql${DCAP_VERSION:+"=${DCAP_VERSION}"} libsgx-dcap-ql-dev${DCAP_VERSION:+"=${DCAP_VERSION}"} &&
-        sudo apt-get install -y libsgx-dcap-quote-verify${DCAP_VERSION:+"=${DCAP_VERSION}"} libsgx-dcap-quote-verify-dev${DCAP_VERSION:+"=${DCAP_VERSION}"}
+        sudo apt-get install -y libsgx-dcap-ql${DCAP_VERSION:+"=${DCAP_VERSION}"} 
+                                libsgx-dcap-ql-dev${DCAP_VERSION:+"=${DCAP_VERSION}"} 
+                                libsgx-dcap-quote-verify${DCAP_VERSION:+"=${DCAP_VERSION}"} 
+                                libsgx-dcap-quote-verify-dev${DCAP_VERSION:+"=${DCAP_VERSION}"} 
+                                libsgx-ae-id-enclave${DCAP_VERSION:+"=${DCAP_VERSION}"} 
+                                libsgx-ae-qe3${DCAP_VERSION:+"=${DCAP_VERSION}"} 
+                                libsgx-ae-qve${DCAP_VERSION:+"=${DCAP_VERSION}"} 
+                                libsgx-qe3-logic${DCAP_VERSION:+"=${DCAP_VERSION}"} 
+                                libsgx-pce-logic${DCAP_VERSION:+"=${DCAP_VERSION}"} 
```

Comment:
All of the dcap library versions had to specified, see https://github.com/mobilecoinfoundation/sgx/actions/runs/5673509866/job/15375128115 for where it was mixing 1.16 and 1.17.